pkg/sql/colexec/external/external.go

```diff
@@ -49,7 +49,7 @@ import (
 )
 
 var (
-	ONE_BATCH_MAX_SIZE = 10 * 1024 * 1024 // 10M
+	ONE_BATCH_MAX_SIZE = int(0.7 * float64(morpc.DefaultMaxMessageSize))
```

Comment:
can not use default rpc size as BATCH size. You need check the config first. If config is 0, than use the default value

Reply:
The batchsize should be determined with the parameters of cn and should not be very large. And batchsize should never be very large.